minio_examples/Dockerfile

```diff
@@ -17,18 +17,26 @@ FROM quay.io/cloudservices/minio:RELEASE.2021-06-17T00-10-46Z.hotfix.35a0912ff a
 
 EXPOSE 9000
 
-RUN useradd -u 1000 -g 0 && mkdir -p /data1/modelmesh-example-models && chown -R 1000:0 /data1
+USER root
 
-COPY --chown 1000:0 sklearn /data1/modelmesh-example-models/sklearn/
-COPY --chown 1000:0 lightgbm /data1/modelmesh-example-models/lightgbm/
-COPY --chown 1000:0 onnx /data1/modelmesh-example-models/onnx/
-COPY --chown 1000:0 pytorch /data1/modelmesh-example-models/pytorch/
-COPY --chown 1000:0 xgboost /data1/modelmesh-example-models/xgboost/
-COPY --chown 1000:0 tensorflow /data1/modelmesh-example-models/tensorflow/
-COPY --chown 1000:0 keras /data1/modelmesh-example-models/keras/
+RUN mkdir -p /data1/modelmesh-example-models
+
+COPY sklearn /data1/modelmesh-example-models/sklearn/
+COPY lightgbm /data1/modelmesh-example-models/lightgbm/
+COPY onnx /data1/modelmesh-example-models/onnx/
+COPY pytorch /data1/modelmesh-example-models/pytorch/
+COPY xgboost /data1/modelmesh-example-models/xgboost/
+COPY tensorflow /data1/modelmesh-example-models/tensorflow/
+COPY keras /data1/modelmesh-example-models/keras/
+
+RUN useradd -u 1000 -g 0 modelmesh
+
+RUN chown -R 1000:0 /data1
```

Comment:
as Nick pointed out, doing a `chown` after the `COPY` layers causes the final Docker image to be about 100 MB larger. Reverting to `COPY --chown 1000:0 ...` in next commit

Reply:
https://hub.docker.com/r/kserve/modelmesh-minio-examples/tags
https://hub.docker.com/r/kserve/modelmesh-minio-dev-examples/tags